### ISSUE TITLE
Bosh create-env fails when release template uses `-%>`

### ISSUE THREAD

**Issue by rkoster** [cloudfoundry/bosh-cli#381]
This difference in behaviour can be trace back to a different ERB uses in the evaluation contexts:
```ruby
# bosh create-env
erb = ERB.new(File.read(src_path))

# director
template = ERB.new(File.read(template_name), safe_level = nil, trim_mode = "-")
```
source [cli](https://github.com/cloudfoundry/bosh-cli/blob/master/templatescompiler/erbrenderer/template_evaluation_context_rb.go#L177), [director](https://github.com/cloudfoundry/bosh/blob/master/src/bosh-template/lib/bosh/template/renderer.rb#L15)

Is this difference in behaviour intended or would a PR to fix it be merged?

**Comment by voelzmo** [comment]
I guess, I only PR'ed trim support for the director code. It was not intentionally left out for the cli. Damn you, double maintenance. ;)

PR for fixing would be appreciated!

**Comment by cppforlife** [comment]
available in v2.0.47+
